lib/config-migrator.js

```diff
@@ -0,0 +1,59 @@
+/** @babel */
+
+import _ from 'lodash'
+import { heredoc } from './werkzeug'
+
+export default async function checkConfigAndMigrate () {
```

Comment:
Now that I think about it, why are we even checking runtime dependencies here? This is supposed to be config related, yeah?

Runtime checks should be can be executed as a separate step. They can be async (but we need to handle resolution/rejection).